Source/JavaScriptCore/jsc.cpp

```diff
@@ -3072,6 +3072,8 @@ JSC_DEFINE_HOST_FUNCTION(functionDropAllLocks, (JSGlobalObject* globalObject, Ca
 #define EXCEPT(x)
 #endif
 
+static LazyNeverDestroyed<BinarySemaphore> waitToExit;
```

Comment:
You can put `static BinarySemaphore waitToExit` (`LazyNeverDestroyed` is not necessary).